### PR TITLE
📖 Improve README.md for the separated TMC placement controller

### PR DESCRIPTION
**Pull request by waltforme**

### CODE REVIEW

File: cmd/placement/README.md [hunk 1]
```diff
@@ -38,6 +38,25 @@ In the `root:compute` workspace, create Location `foo`:
...
+    ├── 2uctu097ej9hv9kv
+    └── 31xvmeuhuk8tykwy
+        └── 1ssldk2xdtia9an1
+```
+However, we can still access the workspaces by their human-readable names, as long as we know the names and their structure.
+```
+👉 edge-mc $ kubectl ws root:compute
+Current workspace is "root:compute".
+👉 edge-mc $ kubectl ws root:my-org:dev
+Current workspace is "root:my-org:dev".
```
Comment (MikeSpreitzer) [inline cmd/placement/README.md:57]: But a typical user's workspace is named differently.  A typical user's workspace would be entered by `kubectl ws \
``.

Comment (waltforme) [reply cmd/placement/README.md:57]: OK. Tried to handle this in force-push https://github.com/kcp-dev/edge-mc/commit/a56946546957d1038255d6a6178c52e4a64b8ca3.

**Review by MikeSpreitzer — APPROVED** [review]
/lgtm

**Pull request merged**